tests/components/GenerativeDirectAnswer.test.tsx

```diff
@@ -0,0 +1,53 @@
+import { render, screen } from '@testing-library/react';
+
```

Comment:
you may also want to add some storybook stories (some of which would be viewable by users of SDK on this [page](https://storybook.hitchhikers.yext.com/?path=/docs/searchbar--docs)).

Reply:
will do in a followup PR